include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -3309,7 +3309,12 @@ struct OrtApi {
   * The function will find the initialized tensors with external in the graph and will replace
   * them with the tensors provided with inputs. The API verifies that the tensor being replaced
   * has external data reference(s), has the same name, dimensions and data type. The replacement
-  * will take place before any of the optimizations take place.
+  * will take place before any of the optimizations take place. The data will be copied into the graph
+  * since TensorProto cant refer to the user provided buffers.
```

Comment:
nit: can't

Reply:
The spell check for both Studio and MS Outlook says cant is the word to use.

Reply:
while it is a word I think we want "can't" here
[https://www.merriam-webster.com/dictionary/cant](https://www.merriam-webster.com/dictionary/cant)